docs/supported-libraries.md

```diff
@@ -19,119 +19,123 @@ These are the supported libraries and frameworks:
+| [Vert.x Kafka Client](https://vertx.io/docs/vertx-kafka-client/java/)                                                                       | 3.6+                          | N/A                                                                                                                                                                                                                                                                                                                                                                                     | [Messaging Spans]                                                                      |
+| [Vert.x RxJava2](https://vertx.io/docs/vertx-rx/java2/)                                                                                     | 3.5+                          | N/A                                                                                                                                                                                                                                                                                                                                                                                     | context propagation only                                                               |
+| [Vibur DBCP](https://www.vibur.org/)                                                                                                        | 11.0+                         | [opentelemetry-vibur-dbcp-11.0](../instrumentation/vibur-dbcp-11.0/library)                                                                                                                                                                                                                                                                                                             | [Database Pool Metrics]                                                                |
 
 **[1]** Standalone library instrumentation refers to instrumentation that can be used without the Java agent.
 
+**[2]** Provides `http.route`: Provides route-based span name for existing `SERVER` span. If applicable, provides `http.route` span and metric attribute on existing `SERVER` span and metrics.
+
+**[3]** Controller Spans are `INTERNAL` spans capturing the controller and/or view execution. See [Suppressing controller and/or view spans]](https://opentelemetry.io/docs/instrumentation/java/automatic/agent-config/#suppressing-controller-andor-view-spans).
```

Comment:
```suggestion
**[3]** Controller Spans are `INTERNAL` spans capturing the controller and/or view execution and follow [Source Code Attributes] semantic conventions. See [Suppressing controller and/or view spans]](https://opentelemetry.io/docs/instrumentation/java/automatic/agent-config/#suppressing-controller-andor-view-spans).
```

Reply:
Is this true of all controller spans?

Reply:
it doesn't appear to be (I think it should be true for controller spans, though not for view spans, I'll open an issue)

probably makes sense to leave this note out of the docs for now until we update instrumentation and tests to confirm

Reply:
(#7345)